docs/database/database-spans.md

```diff
@@ -76,6 +77,17 @@ and SHOULD adhere to one of the following values, provided they are accessible:
 If a corresponding `{target}` value is not available for a specific operation, the instrumentation SHOULD omit the `{target}`.
 For example, for an operation describing SQL query on an anonymous table like `SELECT * FROM (SELECT * FROM table) t`, span name should be `SELECT`.
 
+## Sanitization of `db.query.text`
+
+Sanitization MUST be done by replacing all literals with a placeholder value. Such literals
```

Comment:
It could be interpreted as that sanitization is a MUST and I hope we can clarify it with something like

```suggestion
The `db.query.text` SHOULD be collected by default only if there is sanitization that excludes sensitive information.

Sanitization SHOULD replace all literals with a placeholder value. Such literals
```

I intentionally changed MUST to SHOULD since the sanitization so far meant removing sensitive information, not all literals. Happy to discuss more.

But also it feels that we use MUST to tell **how** to do sanitization and it's too strong.

Reply:
`SHOULD` sounds ok to me here as well. I was going back and forth between `SHOULD` and `MUST` - so let's just move on with `SHOULD`.